tests/framework/installer/ceph_manifests.go

```diff
@@ -1028,6 +1028,12 @@ metadata:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: rook-ceph-default
+  namespace: ` + namespace + `
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
can we do this? directly add a new service account instead of putting adding in middle
```
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: rook-ceph-default
  namespace: ` + namespace + `
---
```